src/Interpreters/Cache/FileCache.cpp

```diff
@@ -889,6 +896,9 @@ bool FileCache::tryReserve(
     if (!main_priority->collectCandidatesForEviction(
             size, required_elements_num, reserve_stat, eviction_candidates, queue_iterator, user.user_id, cache_lock))
     {
+        LOG_DEBUG(log, "Collection candidates for eviction from cache is unsuccessful, "
```

Comment:
I do not think it is worth it, this will pollute the logs unnecessarily.

Reply:
Is this about this specific LOG_DEBUG or the whole PR?

Reply:
@yakov-olkhovskiy, replace with `LOG_TEST` and it will be good to merge.

Reply:
@alexey-milovidov we need this diagnostics in CI runs (at least temporarily - until we figure out the problem) - and CI as I remember don't run at TEST log level. If only this particular log then maybe ok - we will be able to figure it out by elimination.
@kssenii is it ok to convert only this one to LOG_TEST and left others at LOG_DEBUG?